src/scalmat.jl

```diff
@@ -53,6 +53,7 @@ function \(a::ScalMat, x::AbstractVecOrMat)
 end
 function /(x::AbstractVecOrMat, a::ScalMat)
     @check_argdims a.dim == size(x, 2)
+    VERSION >= v"1.9" && return x .* inv(a.value)
```

Comment:
Same as above, use the exact version number?

Reply:
Reverted for compliance.